knexfile.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
   test: defaults,
   development: defaults,
   dummy: Object.assign({}, defaults, { seeds: { directory: './seeds/dummy' } }),
+  farmdev: Object.assign({}, defaults, { seeds: { directory: './seeds/farm-dev' } }),
```

Comment:
would love to figure out a better seeding system. why not always include a dummy farm or two in our seeds? but nothing to do for now